keyboards/split75/config.h

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#define VENDOR_ID       0x20A0
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Wheatfield
+#define PRODUCT         Split75   
+#define DESCRIPTION     VE.A Clone
+
+/* matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 14
+
+#define RGBLED_NUM 8
+#define RGBLIGHT_ANIMATIONS
```

Comment:
```suggestion
#define RGBLIGHT_ANIMATIONS

#define BACKLIGHT_PIN D4
```

This PCB looks to have backlight support. The reference circuit for split PS2AVRGB boards is here: https://github.com/showjean/ps2avrU/blob/master/firmware/ps2avrGB/Circuit/ps2avrGB_split/20150909_013330_001.png

Caps lock is on D1, Num lock on D0, and Scroll lock on D6 (the same as non-split PS2AVRGB).

Reply:
It does, currently the backlight is always on for this keyboard